algorithms/Lesson4_CountingElements/ex_2.rb

```diff
@@ -0,0 +1,23 @@
+def solution(x, array)
+  n = array.index(x) if array.include? x
```

Comment:
remember that `.index(x)` is `O(n)`, i.e. pessimistic case is that it needs to scan the whole array to find an element.